include/utils/MastodonUtils.h

```diff
@@ -40,6 +42,84 @@ std::vector<std::vector<Real>> responseSpectrum(const Real & freq_start,
 std::vector<std::vector<Real>> regularize(const std::vector<Real> & history_acc,
                                           const std::vector<Real> & history_time,
                                           const Real & reg_dt);
+
+/**
+ * Function to check if all the vectors in a vector are of equal size
+ */
+bool checkEqualSize(const std::vector<std::vector<Real>> & vectors);
```

Comment:
Should this be "areEqualSize" or should "areEqual" be "checkEqual", I think I would prefer "areEqual" -> "checkEqual"

Reply:
Fair point. I changed them to checkEqual and checkEqualSize.